.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
+min_python_version = 3.7
```

Comment:
Since Astro only supports Python 3.7, I've added this to avoid:
```
src/astro/utils/__init__.py:1:1: TYP001 guard import by `if False:  # TYPE_CHECKING`: Type (not in 3.5.0, 3.5.1)
```
More info:
https://github.com/asottile/flake8-typing-imports